go/ct/rlz/conditions.go

```diff
@@ -328,8 +328,9 @@ func IsRevision(revision Revision) Condition {
 	return RevisionBounds(revision, revision)
 }
 
+// Any revision covered by the CT, excludes future revisions.
```

Comment:
nit: comments in Go by convention always start with the function or type documented. In this case
```suggestion
// AnyKnownRevision restricts the revision to any revision covered by the CT specification.
```